infra/cardano-sl-infra.cabal

```diff
@@ -164,6 +164,7 @@ library
                       , universum
                       , unliftio
                       , unordered-containers
+                      , vector
```

Comment:
We can avoid adding a dep on vector by using `Universum`'s polymorphic `toList` [here](https://github.com/input-output-hk/cardano-sl/pull/3354/files#diff-4f54fc76b27da1d0aaf520982bffb13dR49). Not a big deal, but it makes the commits / PR cleaner.